pkg/pool-manager/pool_test.go

```diff
@@ -414,4 +416,56 @@ var _ = Describe("Pool", func() {
 		})
 	})
 
+	Describe("Multus Network Annotations API Tests", func() {
+		Context("when pool-manager is configured with available addresses, Pod is created with networks annotation with mac-address", func() {
```

Comment:
This is missleading. You say that "Pod is created", but it is not created in the context (before each), it is created only in the describe table only. You can drop the variable from here and keep it in DescribeTable. You should also remove the "Pod is created" from this title.

Reply:
@ormergi you haven't dropped the `pod` initialization from the context. It is no longer needed there

Reply:
done